d2/src/test/java/com/linkedin/d2/xds/TestXdsToD2PropertiesAdaptor.java

```diff
@@ -74,26 +77,27 @@ public void testListenToNormalCluster()
     XdsToD2PropertiesAdaptorFixture fixture = new XdsToD2PropertiesAdaptorFixture();
     fixture.getSpiedAdaptor().listenToCluster(PRIMARY_CLUSTER_NAME);
 
-    verify(fixture._xdsClient).watchXdsResource(eq(PRIMARY_CLUSTER_RESOURCE_NAME), eq(XdsClient.ResourceType.D2_NODE), any());
+    verify(fixture._xdsClient).watchXdsResource(eq(PRIMARY_CLUSTER_RESOURCE_NAME), eq(XdsClient.ResourceType.NODE), any());
     verifyClusterNodeUpdate(fixture, PRIMARY_CLUSTER_NAME, null, PRIMARY_CLUSTER_PROPERTIES);
   }
 
   @Test
-  public void testListenToClusterSymlink() {
+  public void testListenToClusterSymlink()
+  {
     XdsToD2PropertiesAdaptorFixture fixture = new XdsToD2PropertiesAdaptorFixture();
     fixture.getSpiedAdaptor().listenToCluster(SYMLINK_NAME);
 
-    verify(fixture._xdsClient).watchXdsResource(eq(CLUSTER_SYMLINK_RESOURCE_NAME), eq(XdsClient.ResourceType.D2_SYMLINK_NODE), any());
+    verify(fixture._xdsClient).watchXdsResource(eq(CLUSTER_SYMLINK_RESOURCE_NAME), eq(XdsClient.ResourceType.NODE), any());
 
-    XdsClient.D2SymlinkNodeResourceWatcher symlinkNodeWatcher =
-        (XdsClient.D2SymlinkNodeResourceWatcher) fixture._symlinkWatcherArgumentCaptor.getValue();
+    XdsClient.SymlinkNodeResourceWatcher symlinkNodeWatcher =
+        (XdsClient.SymlinkNodeResourceWatcher) fixture._clusterWatcherArgumentCaptor.getValue();
```

Comment:
Why do we change `fixture._symlinkWatcherArgumentCaptor` to `fixture._clusterWatcherArgumentCaptor` here?

Reply:
Ah, I see, we removed `_symlinkWatcherArgumentCaptor` since symlinks and Nodes can be treated the same way under the new scheme

Reply:
This has actually be reverted actualyl